diagnostics/report.go

```diff
@@ -58,8 +58,13 @@ func generateClient(clientUrl string, authEnabled bool) weaviate.Config {
 		return config
 	}
 
-	username := getInput("Username:", ' ')
-	password := getInput("Password:", '*')
+	username := globalConfig.User
+	password := globalConfig.Pass
+
+	if username == "" {
+		username = getInput("Username:", ' ')
+		password = getInput("Password:", '*')
```

Comment:
Thanks @kcm Can you change this so username can be supplied but password is done via `getInput`?